tests/store/crud/read/test_read_analysis.py

```diff
@@ -466,23 +466,23 @@ def test_get_analyses_for_case_before(
 def test_get_analyses_for_pipeline_before(
     store_with_analyses_for_cases_not_uploaded_fluffy: Store,
     timestamp_now: datetime,
-    pipeline: Workflow = Workflow.FLUFFY,
+    workflow: Workflow = Workflow.FLUFFY,
 ):
     """Test to get all analyses for a pipeline before a given date."""
 
     # GIVEN a database with a number of analyses
 
     # WHEN getting all analyses before a given date
     analyses: list[Analysis] = (
-        store_with_analyses_for_cases_not_uploaded_fluffy.get_analyses_for_pipeline_started_at_before(
-            started_at_before=timestamp_now, pipeline=pipeline
+        store_with_analyses_for_cases_not_uploaded_fluffy.get_analyses_for_workflow_started_at_before(
+            workflow=workflow, started_at_before=timestamp_now
         )
     )
 
     # THEN assert that the analyses before the given date are returned
     for analysis in analyses:
         assert analysis.started_at < timestamp_now
-        assert analysis.pipeline == pipeline
+        assert analysis.pipeline == workflow
```

Comment:
```suggestion
        assert analysis.workflow == workflow
```

Reply:
This is in the DB model - not in this PR